.github/workflows/ci-python.yml

```diff
@@ -0,0 +1,27 @@
+---
+name: Python CI
+on: [push, pull_request]
+
+jobs:
+  python-build:
+    name: Python Build
+    runs-on: ubuntu-latest
+    container: fedora:latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install dependencies
+        run: |
+          sudo dnf -y install dnf-plugins-core
+          sudo dnf -y builddep libcomps.spec
+          pip install --upgrade pip
```

Comment:
At this stage, nitpicking :slightly_smiling_face: but is the upgrade call necessary? (Just trying to keep the variance between workflows minimal, we don't have this in createrepo_c, either it's a different case or we can be uniform?)

Reply:
It's not necessary on Fedora, it is necessary on images with older versions of pip.  Without it you need to manually install "skbuild" because pip doesn't know how to read ```pyproject.toml```

With that said, if you would prefer to either drop it or do the manual install, either of those are fine with me.

Reply:
Well, the workflow is running on the latest fedora, so it's always going to work here IIUC? (if so, I'd prefer to drop it)